lib/gui/index.html

```diff
@@ -57,7 +57,8 @@
         isFinish: state.is('success')
       }">
       <safe-webview
-        src="'https://etcher.io/success-banner/'"></safe-webview>
+        src="'https://etcher.io/'"
+        on-state-change="state.onStateChange"></safe-webview>
```

Comment:
It looks weird that the `safe-webview` is exposing this attribute. Can this all be handled internally?